<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="Viewer" name="Canvas"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionViewer_Help"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuProjection">
     <property name="title">
      <string>Projection</string>
     </property>
     <addaction name="actionOrthographic"/>
     <addaction name="actionPerspective"/>
    </widget>
    <addaction name="menuProjection"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_V"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionViewer_Help">
   <property name="text">
    <string>Viewer Help</string>
   </property>
  </action>
  <action name="actionOrthographic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Orthographic</string>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
